db/migrate/20160330040409_fill_confirmed_at_for_comments.rb

```diff
@@ -0,0 +1,5 @@
+class FillConfirmedAtForComments < ActiveRecord::Migration
+  def change
+    Comment.fill_confirmed_at_for_existing_confirmed_comments
```

Comment:
@equivalentideas I just noticed this was an irreversible migration. It's a good idea to always try to make migrations reversible.